lib/orbf/rules_engine/data/contract.rb

```diff
@@ -0,0 +1,52 @@
+module Orbf
+  module RulesEngine
+    class Contract
+      KNOWN_FIELDS = %w[startContract endContract id org_unit].freeze
+      attr_reader :id, :start_period, :end_period, :field_values
+      def initialize(field_values, calendar)
+        @field_values = field_values
+        @id = field_values["id"]
+        @org_unit = field_values["org_unit"]
+        @start_period = field_values["startContract"].gsub("-", "").slice(0, 6)
+        @end_period = field_values["endContract"].gsub("-", "").slice(0, 6)
+        @calendar = calendar
+      end
+
+      def match_period?(period)
+        start_month_period = @calendar.periods(period, "monthly")[0]
```

Comment:
I think I'll need to change this to "some periods" and not only the first period